src/script-templates/caf.in

```diff
@@ -161,14 +161,13 @@ __caf_err_report() {
 trap '__caf_err_report "${FUNCNAME:-.}" ${LINENO}' ERR
 
 # See if we are compiling or compiling and/or linking
-__only_compiling () {
```

Comment:
Just curious @vehre: what was your motivation for changing this from a function to a variable?